commit b3bd21e9f07272a37c2259070f95b167e6301c79
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Mon Jul 14 11:53:10 2008 +0300

    Fix mono dependency extraction
    - use "Mono/.Net assembly" instead of "PE executable" for detection,
      later file reports bitness (eg "PE32 executable"), we just care if its
      mono or not. This is compatible with older libmagic too.

diff --git a/build/rpmfc.c b/build/rpmfc.c
index 3cc2d6d..6f15b2c 100644
--- a/build/rpmfc.c
+++ b/build/rpmfc.c
@@ -490,7 +490,7 @@ static const struct rpmfcTokens_s const rpmfcTokens[] = {
 
   /* XXX .NET executables and libraries.  file(1) cannot differ from win32 
    * executables unfortunately :( */
-  { "PE executable",            RPMFC_MONO|RPMFC_INCLUDE },
+  { "Mono/.Net assembly",       RPMFC_MONO|RPMFC_INCLUDE },
 
   { "current ar archive",	RPMFC_STATIC|RPMFC_LIBRARY|RPMFC_ARCHIVE|RPMFC_INCLUDE },
 
